vms/platformvm/state/state.go

```diff
@@ -547,6 +560,18 @@ func newState(
 		return nil, err
 	}
 
+	subnetManagerDB := prefixdb.New(SubnetManagerPrefix, baseDB)
+	subnetManagerCache, err := metercacher.New[ids.ID, chainIDAndAddr](
+		"subnet_manager_cache",
+		metricsReg,
+		cache.NewSizedLRU[ids.ID, chainIDAndAddr](execCfg.SubnetManagerCacheSize, func(_ ids.ID, f chainIDAndAddr) int {
+			return ids.IDLen + len(f.Addr)
```

Comment:
We typically account for both the key size and the value size.
```suggestion
			return 2*ids.IDLen + len(f.Addr)
```